src/main/java/screens/collaborative_task_scheduling/ScheduleCTController.java

```diff
@@ -3,7 +3,6 @@
 import entities.Task;
```

Comment:
still not sure this follows Clean Architecture but also not sure how to fix it--ask TA